assets/dev/js/editor/elements/views/base.js

```diff
@@ -553,7 +554,9 @@ BaseElementView = BaseContainer.extend( {
 	renderCustomElementID() {
 		const customElementID = this.getEditModel().get( 'settings' ).get( '_element_id' );
 
-		this.$el.attr( 'id', customElementID );
+		if ( customElementID ) {
+			this.$el.attr( 'id', customElementID );
```

Comment:
Not related to this task, this condition will prevent the creation of redundant empty `id` attribute tags in the elements.